bazel/repository_locations.bzl

```diff
@@ -490,6 +490,18 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-07-10",
         cpe = "cpe:2.3:a:nodejs:node.js:*",
     ),
+    com_github_nodejs_llhttp = dict(
+        project_name = "llhttp",
+        project_desc = "Parser for HTTP messages written in C",
+        project_url = "https://github.com/nodejs/llhttp",
+        version = "5.1.0",
+        sha256 = "7140bbafa30f9cad4d20a0065953bd19bcc11203ab9a901ce7dea1b8182c8c78",
+        strip_prefix = "llhttp-release-v{version}",
+        urls = ["https://github.com/nodejs/llhttp/archive/release/v{version}.tar.gz"],
+        use_category = ["dataplane_core"],
+        release_date = "2021-04-06",
+        cpe = "cpe:2.3:a:nodejs:node.js:*",
+    ),
```

Comment:
There is another version bump - https://github.com/nodejs/llhttp/releases/tag/release%2Fv6.0.0

Reply:
Yeah, sorry for that. I've merged pull request for RTSP support without realizing that the exported RTSP methods would interact badly with existing Node.js ecosystem. Had to do a major release because the C headers changed a bit. I don't think there's anything of concern for envoyproxy, but glad that you keep a note of a new release!